store/project_member.go

```diff
@@ -123,35 +123,36 @@ func (s *ProjectMemberService) DeleteProjectMember(ctx context.Context, delete *
 	return nil
 }
 
-// SetProjectMember set the project member with provided project member list
-func (s *ProjectMemberService) SetProjectMember(ctx context.Context, set *api.ProjectMemberSet) (createdMember, deletedMember []*api.ProjectMember, err error) {
+// BatchUpdateProjectMember update the project member with provided project member list
+func (s *ProjectMemberService) BatchUpdateProjectMember(ctx context.Context, batchUpdate *api.ProjectMemberBatchUpdate) (createdMember, deletedMember []*api.ProjectMember, err error) {
 	tx, err := s.db.BeginTx(ctx, nil)
 	if err != nil {
 		return nil, nil, FormatError(err)
 	}
 	defer tx.PTx.Rollback()
 
-	findProjectMember := &api.ProjectMemberFind{ProjectID: &set.ID}
+	findProjectMember := &api.ProjectMemberFind{ProjectID: &batchUpdate.ID}
 	existingProjectMemberList, err := findProjectMemberList(ctx, tx.PTx, findProjectMember)
```

Comment:
We should do a find with a role_provider filtering? Then we don't need to deal with fmt.Sprintf("%v-%v"...) below.